levels.go

```diff
@@ -753,24 +753,46 @@ func hasAnyPrefixes(s []byte, listOfPrefixes [][]byte) bool {
 	return false
 }
 
-func containsPrefix(smallValue, largeValue, prefix []byte) bool {
+func containsPrefix(table *table.Table, prefix []byte) bool {
```

Comment:
Add a couple of tests.
Build a table with some prefixes and then run contains prefix on it.

Make sure there is a test for prefix (with versions) which lies at the end of the table.

Reply:
Done